PSKoans/Koans/Cmdlets 2/AboutOutCmdlets.Koans.ps1

```diff
@@ -115,23 +115,20 @@ Context 'Out-* Cmdlets' {
             Out-String is a fantastic and simple way to turn complex objects into strings that look just
             like PowerShell's default console formatting.
         #>
-
+    
```

Comment:
Looks like we added some whitespace on this line. Best to remove it just for cleanliness' sake. 🙂

Reply:
Done.